sequencer.c

```diff
@@ -766,6 +766,27 @@ N_("you have staged changes in your working tree\n"
 #define VERIFY_MSG  (1<<4)
 #define CREATE_ROOT_COMMIT (1<<5)
 
+static int run_command_silent(struct child_process *cmd)
+{
+	struct strbuf buf = STRBUF_INIT;
+	int rc;
+
+	/* hide stderr on success */
+	cmd->stdout_to_stderr = 1;
+	cmd->err = -1;
```

Comment:
From the comment of pipe_command:

 * The fields of "cmd" should be set up as they would for a normal run_command
 * invocation. But note that there is no need to set the in, out, or err
 * fields; pipe_command handles that automatically.

I don't think you need to set cmd.err here

Reply:
Oh, before this was done to signal if we need to have the silen_on_success invocation, but now it is not needed any more as the signaling is done otherwise.